network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -1213,3 +1298,10 @@ def invalidate_cache(request, page):
     """
     if isinstance(page, ProductPage):
         cache.clear()
+
+
+def get_pni_home_page():
```

Comment:
This function tells wagtail-airtable where to look for a "parent" page. It looks for the first BuyersGuidePage and returns the id.

Reply:
that's a reasonable enough way to find it.